infra/fetch-location-lambda.tf

```diff
@@ -0,0 +1,82 @@
+locals {
+  fetchLocationZipFile = "../lambda-zips/fetch-location-lambda.zip"
+}
+
+variable "MAPBOX_ACCESS_TOKEN" {
+  type        = string
+  description = "Access token of the Mapbox API"
+  sensitive   = true
+}
+
+variable "MAPBOX_GEOCODING_PLACES_API_URL" {
+  type        = string
+  description = "URL of the Mapbox API for geocoding service (using the endpoint 'mapbox.places'). If no value is inserted, 'https://api.mapbox.com/geocoding/v5/mapbox.places' will be used."
+  default     = "https://api.mapbox.com/geocoding/v5/mapbox.places"
+}
```

Comment:
What are your thoughts on creating variables on each file? Do you see a problem long-term to identify what the defined variables are?